code/modules/mob/living/taste.dm

```diff
@@ -15,10 +15,34 @@
 		. = 101 // can't taste anything without a tongue
 
 // non destructively tastes a reagent container
+
+//FermiChem - How to check pH of a beaker without a meter/pH paper.
+//Basically checks the pH of the holder and burns your poor tongue if it's too acidic!
+//TRAIT_AGEUSIA players can't taste, unless it's burning them.
+//taking sips of a strongly acidic/alkaline substance will burn your tongue.
 /mob/living/proc/taste(datum/reagents/from)
+	var/obj/item/organ/tongue/T = getorganslot("tongue")
+	var/taste_sensitivity = get_taste_sensitivity()
+	var/text_output = from.generate_taste_message(taste_sensitivity)
+	if(T)
+		if(istype(T, /obj/item/organ/tongue/cybernetic))
```

Comment:
This could use an organ flag instead of a typecheck.

Reply:
This conflicts with IPC tongues atm, I'm worried IPC players won't be thrilled to find that they're telling everyone about the pH of everything they taste at roundstart.

Reply:
I'll make a note of it for when I follow up with heart and species organ reworks.